src/Parsing/Impl/Parser.cs

```diff
@@ -4527,32 +4527,32 @@ struct Name {
                 if (arg.Name == null) {
                     if (_stubFile || _langVersion >= PythonLanguageVersion.V35) {
                         if (hasKeywordDict) {
-                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, "positional argument follows keyword argument unpacking");
+                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, Resources.PositionalArgumentKeywardArgumentUnpackingErrorMsg);//positional argument follows keyword argument unpacking
                         } else if (keywordCount > 0) {
-                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, "positional argument follows keyword argument");
+                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, Resources.PositionalArgumentKeywardArgumentErrorMsg);//positional argument follows keyword argument
                         }
                     } else if (hasArgsTuple || hasKeywordDict || keywordCount > 0) {
-                        ReportSyntaxError(arg.StartIndex, arg.EndIndex, "non-keyword arg after keyword arg");
+                        ReportSyntaxError(arg.StartIndex, arg.EndIndex, Resources.NonKeywordArgAfterKeywordArgErrorMsg);//non-keyword arg after keyword arg
                     }
                 } else if (arg.Name == "*") {
                     if (hasArgsTuple || hasKeywordDict) {
                         if (!_stubFile && _langVersion < PythonLanguageVersion.V35) {
-                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, "only one * allowed");
+                            ReportSyntaxError(arg.StartIndex, arg.EndIndex, Resources.OnlyOneAllowedErrorMsg.FormatInvariant("*"));//"only one * allowed"
```

Comment:
You can from formatting and include `*` in the resource as is

Reply:
Made recommended changes.